DEVELOPER.md

```diff
@@ -103,6 +103,7 @@ networks:
     prefix: 10.31.83.112/28
     gateway: 1
     dns: 1
+    ntp: 1                    # used to tell accesspoints to use the ntp server of the core router
```

Comment:
We can just make this the default, no?

Reply:
I just mimicked what we have for dns and gateway and I would prefer to have it set and visible in the configuration. On some locations you might even want to change this to another time server e.g. an access point with build in GPS.